pkg/types/metrics.go

```diff
@@ -18,6 +18,7 @@ type Metrics struct {
 type Prometheus struct {
 	Buckets              []float64 `description:"Buckets for latency metrics." json:"buckets,omitempty" toml:"buckets,omitempty" yaml:"buckets,omitempty" export:"true"`
 	AddEntryPointsLabels bool      `description:"Enable metrics on entry points." json:"addEntryPointsLabels,omitempty" toml:"addEntryPointsLabels,omitempty" yaml:"addEntryPointsLabels,omitempty" export:"true"`
+	AddRouterLabels      bool      `description:"Enable metrics on routers." json:"AddRouterLabels,omitempty" toml:"AddRouterLabels,omitempty" yaml:"AddRouterLabels,omitempty" export:"true"`
```

Comment:
Could you change the name of this option into `AddRoutersLabels`?  Just to be consistent with `AddEntryPointsLabels` and `AddServicesLabels`. This is a mistake to have an `s` at the end of router, service and entrypoint but we would rather prefer to keep things consistent and fix them all at once in a separate PR.

Reply:
Sure, no problem